tardis/montecarlo/montecarlo_numba/numba_interface.py

```diff
@@ -352,12 +358,21 @@ def configuration_initialize(runner, number_of_vpackets):
             u.Hz, equivalencies=u.spectral()
         ).value
     )
+    # logger.debug(
+    #     f"Virtual Packet Spawn Start Frequency = {montecarlo_configuration.v_packet_spawn_start_frequency}"
+    # )
```

Comment:
Either remove the comment characters or remove the message. This message seems useful.

Reply:
Uncommented this line as it would be useful bit of information